recipes-qt/qt5/qtbase/CVE-2024-39936-qtbase-5.15.patch

```diff
@@ -0,0 +1,136 @@
+diff --git a/src/network/access/qhttp2protocolhandler.cpp b/src/network/access/qhttp2protocolhandler.cpp
```

Comment:
The patch file should be generated by `git format-patch` (devtool can do this as well) so that it'll have author info, commit description, and the description should have Upstream-Status, etc.

Reply:
Just noticed that the patch was downloaded out of tree.. if we're not able to get the patch from git tree itself, I'm not sure if the right approach is to commit the patch with our info and mention upstream patch or send the patch as is without author info, etc. Do you know what was done for other similar patches?